README.md

```diff
@@ -3,6 +3,16 @@
 This is the Qibo's website repository.
 Here we describe its contents and show you how to run the site in developement mode, in order to collaborate with us.
 
+### Repo's structure
```

Comment:
I suggest to add a directory tree, see [here](https://stackoverflow.com/questions/23989232/is-there-a-way-to-represent-a-directory-tree-in-a-github-readme-md). 
In this way, the description is more understandable.